product_docs/docs/migration_portal/3.4.0/04_mp_migrating_database/01_mp_schema_extraction.mdx

```diff
@@ -9,27 +9,27 @@ legacyRedirectsGenerated:
 
 <div id="mp_schema_extraction" class="registered_link"></div>
 
-**Prerequisites**
+## Prerequisites
 
-Before extracting a schema, you must download the latest EDB DDL Extractor script from the Migration Portal `Projects` page or from the link provided in the DDL Extractor guide in the Portal Wiki. The script can be run in SQL Developer or SQL\*Plus. It uses Oracle’s `DBMS_METADATA` built-in package to extract DDLs for different objects under schemas (specified while running the script). The EDB DDL extractor creates the DDL file uploaded to the portal and analyzed for EDB Postgres compatibility.
+Before extracting a schema, you must download the latest EDB DDL Extractor script from the Migration Portal Projects page or from the link provided in the DDL Extractor guide in the Portal wiki. You can run the script in SQL Developer or SQL\*Plus. It uses Oracle’s `DBMS_METADATA` built-in package to extract DDLs for different objects under schemas, which you specify while running the script. The EDB DDL extractor creates the DDL file uploaded to the portal and analyzed for EDB Postgres compatibility.
 
 !!!Note
     You must have `CONNECT` and `SELECT_CATALOG_ROLE` roles and `CREATE TABLE` privilege.
 
-**For SQL\*Plus**
+## For SQL\*Plus
 
 1.  Connect to SQL\*Plus and run the command:
 
     `SQL>@edb_ddl_extractor.sql`
 
-2.  Provide the schema name and the path or directory in which the extractor will store the extracted DDL. When extracting multiple schemas, use a comma (`‘,’`) as a delimiter.
+2.  Provide the schema name and the path or directory in which the extractor stores the extracted DDL. When extracting multiple schemas, use a comma (`‘,’`) as a delimiter.
 
 !!!Note
-    If you want to extract all the user schemas from the current database, do not mention any schema names while extracting. However, it is recommended to mention the schema names that you would like to extract.
+    If you want to extract all the user schemas from the current database, don't mention any schema names while extracting. However, we recommend that you mention the schema names that you want to extract.
 
 3.  If you want to extract dependent objects from other schemas, enter `yes` or `no`.
 
-    **For example, on Linux:**
+## On Linux
```

Comment:
Not sure if this should be a heading?

Reply:
Thanks. Dee Dee and I looked at this together and came up with a solution.